projects/bp-gallery/cypress/e2e/archives.cy.ts

```diff
@@ -55,16 +55,16 @@ describe('Archives View', () => {
     });
     cy.contains('Link hinzufügen').click();
     cy.get('#archive-form-title').should('be.visible').type('Test-Link 2');
-    cy.get('#archive-form-url').should('be.visible').type('test2.de');
+    cy.get('#archive-form-url').should('be.visible').type('test2.de').focus().blur();
     cy.get('.archive-link-entry')
       .first()
       .within(() => {
         return cy.get('[data-testid="EditIcon"]').click();
       });
-    cy.get('#archive-form-title').should('be.visible').type(' Edit');
+    cy.get('#archive-form-title').should('be.visible').type(' Edit').focus().blur();
     cy.contains('Link hinzufügen').click();
     cy.get('#archive-form-title').should('be.visible').type('Test-Link 3');
-    cy.get('#archive-form-url').should('be.visible').type('test3.de');
+    cy.get('#archive-form-url').should('be.visible').type('test3.de').focus().blur();
```

Comment:
Can you explain why you focus and unfocus the inputs?

Reply:
I have found [this]. But I don't know why the problem should appear after switching to vite.

I think the `.focus().blur()` is slightly confusing. Have you tried these other options?:
- `type('...{enter}');`
- `.blur();` only
- `.focus().type('...').blur()`

I ordered them by what I think is most readable (most readable first). If none of them work, I think a comment that explains why the `.focus().blur()` is needed would be appropriate.

[this]: https://stackoverflow.com/questions/73221024/cypress-blur-events-when-input-loses-focus-are-not-triggered-with-headless-cy